droidlet/dashboard/web/src/StateManager.js

```diff
@@ -288,6 +293,20 @@ class StateManager {
       if (ref instanceof MainPane) {
         ref.setState({ agentType: this.memory.agentType });
       }
+      if (ref instanceof InteractApp) {
+        ref.setState({ agentType: this.memory.agentType });
+      }
+    });
+  }
+
+  checkIsTurk(status) {
```

Comment:
nit: better naming of function name to reflect the functionality ?

Reply:
changed to 'forceErrorLabeling'